schema.graphql

```diff
@@ -27,26 +26,89 @@ type CustomerBooking {
   departure: RouteStop
   destinationImageUrl(dimensions: BookingDestinationImageDimensions = _600x600): String
   passengerCount: Int
+
+  """OneWay, multicity or return"""
```

Comment:
The casing is weird, is there a way to fix it?

Reply:
Yes, change the description field of the type 😄